cloudbuild.yaml

```diff
@@ -1,27 +0,0 @@
-steps:
-  - name: python:3.10.4-buster
-    id: Python Env Setup And App Tests
-    entrypoint: bash
```

Comment:
As part of this we should be disabling/deleting the trigger that's in the gcp project it's running in